src/index.html

```diff
@@ -2,16 +2,186 @@
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="Name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            How old are you?
+            <input type="number"
+              name="Age"
+              max="100"
+              min="1"
+              value="12"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the aggrement
+            <input
+              type="checkbox"
+              name="acceptTermOfAggrement"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fields">
+        <legend>Registration</legend>
+        <div class="inputs">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="eMail: "
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Password:
+            <input
+              type="password"
+              name="Password"
+              required
+              minlength="5"
+              maxlength="32"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fields">
+        <legend>An interesting fact about you!</legend>
+        <div class="inputs">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label >
+            <input
+              type="radio"
+              name="loveCats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            What is your favorite color?
+            <input type="color" name="favouriteColor: ">
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="timeToGoToBed: "
+              step="1"
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+            What are your favorite brand of cars?
+            <select name="favourite cars: " multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
```

Comment:
better pass lower-cased values to server
```suggestion
              <option value="bmw">BMW</option>
              <option value="audi">Audi</option>
              <option value="lada">Lada</option>
```